skiko/src/jvmMain/kotlin/org/jetbrains/skiko/Library.kt

```diff
@@ -144,6 +140,9 @@ object Library {
 
         val dataDir = File(File(SkikoProperties.dataPath), hash)
         dataDir.mkdirs()
+
+        launchSkikoDataDirCleanupIfNecessary(dataDir.toPath())
```

Comment:
Can we move this to the end of the function? I plan to extract all the `load()` code to an utility function which will be used in 2 places by:
- [Skiko binary loader](https://github.com/JetBrains/skiko/pull/1115/files#diff-04c01299138a9ca5caa241b23b20a92721fccfec9be4f85b280a95e44bb17e11R7)
- [Additional Angle binary loader](https://github.com/JetBrains/skiko/pull/1115/files#diff-4dfc77a2d3233349d447cc9903c015fefbf29d5fe00f615d9a5a6161b4c1f241R9)